### ISSUE TITLE
Fix an issue with mintermization of constants (true/false)

### ISSUE THREAD

**Issue by jurajsic** [VeriFIT/mata#106]
Constants that can occur in automata are not handled right now during mintermization.
For example , for this NFA:
```
@NFA-bits
%States-enum q r s
%Alphabet-auto
%Initial q
%Final r
q true r
r a1 & a2 s
s false s
```
the resulting mintermized NFA should have transitions from `q` to `r` for all minterms (created from the transition `q true r`), while there should be no transitions created from `s false s`.

For AFA, taking this example:
```
@AFA-bits
%Initial q0
%Final t
q0 (true & q2 & q3 & q0) | (a4 & !a5 & !a6 & !a7 & q0 & q1 & q2)
q1 false
q2 q1
q3 true
```
There will be two minterms: `a4 & !a5 & !a6 & !a7` and its negation. Then, from `q0` there should be two transitions (one for each minterm) in mintermized automaton going to `q2 & q3 & q0` created from `(true & q2 & q3 & q0)` and one transition going to `q0 & q1 & q2` with minterm representing `a4 & !a5 & !a6 & !a7`. From `q1` there should be no transitions; the transition `q2 q1` has the same meaning as `q2 true & q1` and from `q3` there will be two transitions (again, one for each minterm) going to `true`.

**Comment by martinhruska** [comment]
Do we consider also the following cases:
1) `q0 true & a1 & !a3 & q2 & q3`
and
2) `q0 a1 & true`, i.e., `true` is used in state part of formula.

**Comment by martinhruska** [comment]
PR is addressed by #108
